### ISSUE TITLE
Disable add-to-cart button when add to an existing subscription checkbox is checked

### ISSUE THREAD

**Issue by thenbrent** [Prospress/woocommerce-subscribe-all-the-things#354]
As reported in https://github.com/Prospress/woocommerce-subscribe-all-the-things/issues/349#issuecomment-426445804

> One thing that would make it less confusing is to disable the add to cart button when they choose to add to an existing subscription.

I _think_ that could work.

Disabling the add-to-cart button might get confusing if the customer changes their mind and decides they want to add the product to their cart after clicking the _Add to an existing subscription_ checkbox though.

**Comment by franticpsyx** [comment]
> Disabling the add-to-cart button might get confusing if the customer changes their mind and decides they want to add the product to their cart after clicking the Add to an existing subscription checkbox though.

👍 

A checkbox below a button shouldn't disable the button -- even if there's no other elements around. If there are other elements before the add-to-sub checkbox and the button, things can easily get very confusing for the user.

Ticking the checkbox there expresses _interest_ but not necessarily _commitment_. Being merely _interested_ in adding a product to an existing subscription doesn't necessarily mean that I'm no longer interested in adding the product to the cart, after all.

We've tried to mitigate confusion there by using the same button styles and language ("Add" vs "Add to Cart").

We could possibly consider changing the "Add" buttons to read "Add to Subscription", in order to increase the contrast between the two actions. I believe that the button is already part of a template, and there's enough templating/filtering infra in place to facilitate customizations.

Summing up, the role of the checkbox is to reveal an additional flow, not replace the add-to-cart flow. It shouldn't be necessary to un-tick the add-to-sub box in order to click the add-to-cart button.

If more

**Comment by SundayDriver** [comment]
> Ticking the checkbox there expresses interest but not necessarily commitment. Being merely interested in adding a product to an existing subscription doesn't necessarily mean that I'm no longer interested in adding the product to the cart, after all.

I agree.
